backend/routers/issues.py

```
@@ -238,6 +238,55 @@ def upvote_issue(issue_id: int, db: Session = Depends(get_db)):
         message="Issue upvoted successfully"
     )
 
+@router.get("/api/issues/recent", response_model=List[IssueResponse])
+def get_recent_issues(
+    limit: int = Query(10, ge=1, le=50, description="Number of issues to return"),
+    offset: int = Query(0, ge=0, description="Number of issues to skip"),
+    db: Session = Depends(get_db)
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)